packages/next/src/server/dev/middleware-turbopack.ts

```diff
@@ -168,14 +170,14 @@ function createStackFrame(
   return {
     file,
     methodName: searchParams.get('methodName') ?? '<unknown>',
-    line: parseInt(searchParams.get('lineNumber') ?? '0', 10) || 0,
-    column: parseInt(searchParams.get('column') ?? '0', 10) || 0,
+    line: parseInt(searchParams.get('line1') ?? '0', 10) || undefined,
+    column: parseInt(searchParams.get('column1') ?? '0', 10) || undefined,
```

Comment:
Is `0` as default correct here?

Reply:
It will be evaluated to `0` which will result in `undefined` being used. It's a convoluted pattern but I opted for a smaller diff instead. The important part is that we now default to `undefined` instead of `0` which wouldn't make sense in 1-based locations.